addons/sale_timesheet/static/src/scss/project_project.scss

```diff
@@ -0,0 +1,6 @@
+@include media-breakpoint-down(md) {
+    .o_invoicing_rate_text {
+        margin-top: -2rem;
```

Comment:
Some o_kanban_ghost records appears under no grouped kanban views (which is the case inside a notebook),  these taking a bit of vertical space under the kanban element so the text displayed under them is a bit too low.
Best fix is to prevent them from appearing with `display: none;` instead of using the negative margin